### PR TITLE
travis.yml: add python 3.7 support

### PR DESCRIPTION
**Pull request by flokli**
see https://github.com/travis-ci/travis-ci/issues/9815#issue-336465122

### CODE REVIEW

**Comment by flokli** [comment]
@polyzen This still errors out and doesn't seem to pick up xenial:

```
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/python-3.7.tar.bz2
$ curl -sSf -o python-3.7.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.7 archive. The archive may not exist. Please consider a different version.
```

Any idea what's wrong here?

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,13 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
```
Comment (polyzen) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,13 @@ python:
...
+
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
Comment (polyzen) [inline .travis.yml]: Seems `sudo` is unnecessary:
https://travis-ci.org/python-security/pyt/builds/480490837/config

**Comment by flokli** [comment]
@polyzen I was misinterpreting Travis' syntax - thanks for the pointers!

**Pull request merged**